doc/source/reference/random/new-or-different.rst

```diff
@@ -21,7 +21,7 @@ Feature            Older Equivalent     Notes
 ------------------ -------------------- -------------
 `~.Generator`      `~.RandomState`      ``Generator`` requires a stream
                                         source, called a `BitGenerator
-                                        <bit_generators>` A number of these
+                                        <_bit_generators>` A number of these
```

Comment:
this should simply be `BitGenerator`

Reply:
fixed this and many more by building with `SPHINXOPTS=-qn` and looking, among the many improper reference warnings, for ones that involve `random`

Reply:
nice